src/service/packet_router.proto

```diff
@@ -44,10 +45,23 @@ message packet_router_packet_up_v1 {
   float snr = 6;
   region region = 7;
   uint64 hold_time = 8;
-  bytes hotspot = 9;
+  bytes gateway = 9;
   bytes signature = 10;
 }
 
+message packet_router_packet_report_v1 {
+  uint64 gateway_timestamp_ms = 1;
+  uint32 oui = 2;
+  uint64 net_id = 3;
+  uint32 rssi = 4;               // LoRa signal strength
+  float frequency_mhz = 5;
+  float snr = 6;
+  data_rate datarate = 7;
+  region region = 8;
+  bytes gateway = 9;
+  bytes lora_payload_hash = 10;  // Hash of `payload` within `message packet`
```

Comment:
remove to payload_hash.. otherwise endless arguments as to whether it's lora or lorawan payload :-/